scripts/get_hashsets.js

```diff
@@ -50,6 +68,20 @@ function handleBreachesResponse(error, response, body) {
   try {
     const breachesJSON = JSON.parse(body);
 
+    s3.listObjects({Bucket: BREACH_HASHSET_BUCKET_NAME}, (err, data) => {
+      if (err) {
+        console.error(`listObjects, err: ${err}`);
```

Comment:
I don't know enough about S3, but should either of these errors (here and L79 below) be fatal? Should we be rethrowing the errors to the outer `catch(err) {...}`?